### PR TITLE
feat: Token expanded state should show additional asset data on L2s/sidechains

### PR DESCRIPTION
**Pull request by welps**
Fixes BACK-286

## What changed (plus any additional context for devs)
The token expanded state will now show the following data if availablle for tokens on L2/sidechains:
- 24h volume
- Market cap
- Description
- Socials

## What to test
The code I touched only affects token expanded state so look at tokens on different networks.

## Final checklist

- [x] Assigned individual reviewers?
- [ ] Added labels? (`critical path`, `release`, `dev QA`)
- [ ] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: src/handlers/dispersion.ts [hunk 1]
```diff
@@ -82,9 +83,31 @@ export const getTrendingAddresses = async (): Promise<
...
+    case Network.optimism:
+      chainId = 10;
+      break;
+    case Network.polygon:
+      chainId = 137;
+      break;
+    default:
+      chainId = 1;
+      break;
+  }
```
Comment (derHowie) [inline src/handlers/dispersion.ts:105]: https://github.com/rainbow-me/rainbow/blob/develop/src/utils/ethereumUtils.ts#L320

We've got a util for this.

**Review by derHowie — COMMENTED** [review]
lgtm, just 1 comment

File: src/handlers/dispersion.ts (hunk 1 above)
Comment (welps) [reply src/handlers/dispersion.ts]: Thanks, fixed!

**Review by derHowie — APPROVED** [review]

**Review by skylarbarrera — APPROVED** [review]
🔥

**Pull request merged**